extensions/ql-vscode/src/run-remote-query.ts

```diff
@@ -128,6 +132,33 @@ export async function runRemoteQuery(cliServer: cli.CodeQLCliServer, credentials
     void showAndLogInformationMessage(`Successfully scheduled runs. [Click here to see the progress](https://github.com/${OWNER}/${REPO}/actions).`);
 
   } catch (error) {
-    void showAndLogErrorMessage(error);
+    if (typeof error.message === 'string' && error.message.includes('Some repositories were invalid')) {
+      const invalidRepos = error?.response?.data?.invalid_repos || [];
+      const reposWithoutDbUploads = error?.response?.data?.repos_without_db_uploads || [];
+      void logger.log('Unable to run query on some of the specified repositories');
+      if (invalidRepos.length > 0) {
+        void logger.log('Invalid repos: ' + invalidRepos.join(', '));
```

Comment:
As Andrew mentioned, we mostly use template literals:
```suggestion
        void logger.log(`Invalid repos: ${invalidRepos.join(', ')}`);
```

(also applies to the other strings below 🙂)

Reply:
Thanks. I changed to backticks but forgot to use the templating 🤦